### ISSUE TITLE
can not get val_acc above 90%

### ISSUE THREAD

**Issue by wangdelp** [raghakot/keras-resnet#35]
Hi there, 

In the paper the val_acc could reach more than 93%, but my trained model using the default configuration could at most got 87% val_acc. I've tried different learning rate schedules but do not improve a lot. 

Any idea how can we further boost the val_acc? Thank you.

**Comment by raghakot** [comment]
I don't think all the subtleties in section 4.2 are implemented (https://arxiv.org/pdf/1512.03385.pdf)
This is just a sample application. Feel free to submit a PR.

It would be good to have a working sample that matches the accuracy described in the paper.
